libdevcore/Common.cpp

```diff
@@ -31,7 +31,9 @@ namespace dev
 
 char const* Version = ETH_PROJECT_VERSION;
 
+const bytes NullBytes;
```

Comment:
While we're here let's change it to `bytes const` according to coding standards.
Nest lines, too.